### ISSUE TITLE
Tensorflow >1.10.1 has warnings

### ISSUE THREAD

**Issue by previ** [CoderBotOrg/backend#95]
When starting coderbot server, importing tensorflow gives the following warnings:
```
/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: compiletime version 3.4 of module 'tensorflow.python.framework.fast_tensor_util' does not match runtime version 3.5
  return f(*args, **kwds)
/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: builtins.type size changed, may indicate binary incompatibility. Expected 432, got 412
  return f(*args, **kwds)
```
It seems to be just a warning (training a new model and using it apparently runs fine)

**Comment by previ** [comment]
fixed moving to tensorflow lite 2.1

**Comment by previ** [comment]
fixed in 4.0/buster image/python 3.7.x
